packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -417,11 +418,19 @@ export function createElement(
         type,
       );
     }
-
     if (type === 'script') {
       // Create the script via .innerHTML so its "parser-inserted" flag is
       // set to true and it does not execute
       const div = ownerDocument.createElement('div');
+      if (__DEV__) {
+        warningWithoutStack(
```

Comment:
Please deduplicate this warning with a boolean so it only ever fires once. (There's some examples in the codebase.)

This should also probably be `warning` rather than `warningWithoutStack`.